components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -4255,6 +4256,69 @@ public static void jsFunction_updateAccessAllowDomains(Context cx, Scriptable th
     }
 
     /**
+     * Given a name of a user the function returns the roles of the user
+     *
+     * @param cx
+     * @param thisObj
+     * @param args
+     * @param funObj
+     * @return
+     * @throws APIManagementException
+     * @throws AxisFault
+     */
+    public static NativeObject jsFunction_getRolesOfUser(Context cx, Scriptable thisObj,
+                                                         Object[] args,
+                                                         Function funObj) throws APIManagementException, AxisFault {
+        String userName = (String) args[0];
+        NativeObject result = new NativeObject();
+        NativeArray roles = new NativeArray(0);
+        boolean isTenantFlowStarted=false;
+        if (userName != null) {
+            String tenantDomain = MultitenantUtils.getTenantDomain(userName);
+            try {
+                if (tenantDomain != null && !MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
+                    // To resolve the caching issue. Otherwise, if the roles of the user were updated, they will not be available here.
+                    isTenantFlowStarted = true;
+                    PrivilegedCarbonContext.startTenantFlow();
+                    PrivilegedCarbonContext.getThreadLocalCarbonContext().setTenantDomain(tenantDomain, true);
+                }
+                int tenantId = ServiceReferenceHolder.getInstance().getRealmService()
+                                                 .getTenantManager()
+                                                 .getTenantId(tenantDomain);
+                org.wso2.carbon.user.api.UserStoreManager manager = ServiceReferenceHolder.getInstance().
+                        getRealmService().getTenantUserRealm(tenantId)
+                                                                 .getUserStoreManager();
+
+                String userNameWithoutTenant=MultitenantUtils.getTenantAwareUsername(userName);
```

Comment:
spaces before and after equls are missing. I noticed this in some places below as well. Simply format the code using the wso2 code formatter :).